File: leetcode/Accepted/count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        count = 0
        for word in words:
            flag = True
            for char in word:
                if char not in allowed:
                    flag = False
                    break

            if flag is True:
                count += 1

        return count

File: leetcode/Accepted/Easy/count-the-number-of-consistent-strings.cpp
class Solution {
 public:
  int countConsistentStrings(string allowed, vector<string>& words) {
    int count = 0;
    for (string word : words) {
      bool flag = true;
      for (char ch : word) {
        if (allowed.find(ch) == string::npos) {
          flag = false;
          break;
        }
      }
      count = flag ? count + 1 : count;
    }
    return count;
  }
};
